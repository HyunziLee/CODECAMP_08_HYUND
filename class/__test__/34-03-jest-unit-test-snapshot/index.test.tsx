import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import JestUnitTestPage from "../../pages/34-03-jest-unit-test-snapshot";

it("기존 사진이랑 비뀐게 없는지 비교해보자-스냅샷테스트", () => {
  const result = render(<JestUnitTestPage />);
  expect(result.container).toMatchSnapshot();
});
