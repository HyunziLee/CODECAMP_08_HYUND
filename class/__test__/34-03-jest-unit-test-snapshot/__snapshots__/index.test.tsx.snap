// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`기존 사진이랑 비뀐게 없는지 비교해보자-스냅샷테스트 1`] = `
<div>
  <div>
    <div>
      영희는 13살
    </div>
    영희 취미: 
    <input
      type="text"
    />
    <button>
      영희랑 놀러가기
    </button>
  </div>
</div>
`;
